src/graphql/schema/collection/createCollectionGqlType.ts

```diff
@@ -45,12 +46,17 @@ export default function createCollectionGqlType<TValue> (
       Array.from(type.fields).map(
         <TFieldValue>([fieldName, field]: [string, ObjectType.Field<TValue, TFieldValue>]) => {
           const { gqlType, intoGqlOutput } = getGqlOutputType(buildToken, field.type)
+          const formattedFieldName = formatName.field(fieldName)
           return {
-            key: formatName.field(fieldName),
+            key: formattedFieldName
             value: {
               description: field.description,
               type: gqlType,
-              resolve: (value: TValue): mixed => intoGqlOutput(field.getValue(value)),
+              sqlName: () => fieldName,
+              sqlExpression: (aliasIdentifier) => sql.query`${sql.identifier(aliasIdentifier)}.${sql.identifier(fieldName)}`,
+              resolve: (value) => {
+                return value.get(fieldName)
```

Comment:
This looks different than the code that was there, `resolve: (value: TValue): mixed => intoGqlOutput(field.getValue(value))`. Will I understand why as I continue? 😊